### PR TITLE
adding entry for CSS clip-path

### PR DESCRIPTION
**Pull request by chrisdavidmills**

### CODE REVIEW

**Comment by marco-c** [comment]
The Travis build is failing, I bet there's some forbidden character

**Comment by chrisdavidmills** [comment]
Hrm, the Travis build is failing again; I really don't know what I've done wrong in my entry.

File: features/css-clip-path.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: "CSS clip-path"
+category: css
+bugzilla: 1247229
+firefox_status: 54
+mdn_url: https://developer.mozilla.org/Web/CSS/clip-path
+spec_url: https://drafts.fxtf.org/css-masking-1/#the-clip-path
+caniuse_ref: clip-path
```
Comment (marco-c) [inline features/css-clip-path.md]: The issue is that this feature doesn't exist on caniuse. Perhaps you meant `css-clip-path`?

Comment (marco-c) [inline features/css-clip-path.md]: (I've now improved the error message when a feature doesn't exist on caniuse)

**Comment by marco-c** [comment]
@chrisdavidmills ping?

**Comment by marco-c** [comment]
The `caniuse_ref` property is wrong (`clip-path` doesn't exist, see http://caniuse.com/#feat=clip-path). Did you mean `css-clip-path`? (http://caniuse.com/#feat=css-clip-path)

**Comment by chrisdavidmills** [comment]
D'oh, sorry - I missed this comment previously. Fixed now!

**Comment by marco-c** [comment]
Thanks ;)

**Pull request merged**